engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeStdlibTest.scala

```diff
@@ -245,6 +252,28 @@ class RuntimeStdlibTest
       ("Standard", "Base", TestEdition.testLibraryVersion.toString)
```

Comment:
If I understand correctly this test checks parsing of docs of the `Standard.Base` library.

Do you think it would be feasible to have some, possibly optional, test which could be run on CI and parse docstrings of all of our libraries?

After all the bug that sparked this fix has appeared in `Standard.Table`, so if I understand correctly this test would not catch it anyway?

I think it is quite important for CI to be able to help us avoid situations like we had where we've found that the LS hangs if `Standard.Table` is loaded in interactive mode.

Reply:
For some reason I thought that this test is running on the whole stdlib. Fixed